src/components/JTextArea/jTextArea.m.scss

```diff
@@ -0,0 +1,43 @@
+@import 'styles/mixins';
+
+.core {
+  @include hover-animation;
+  @include font($form-field-white-font-size);
+
+  position: relative;
+  width: 100%;
+  min-width: 100%;
+  border-radius: $form-border-radius;
+  line-height: $form-field-line-height;
+  resize: none;
+  border-width: 1px;
+  border-style: solid;
+  padding: $form-field-white-padding-vertical $form-field-white-horizontal-padding;
+  font-weight: 600;
+
+  // stylelint-disable selector-nested-pattern, rscss/class-format
+  &.blue {
```

Comment:
can we make plain structure in css modules?
to avoid selectors like `.with.error`